rllib/utils/tracking_dict.py

```diff
@@ -17,6 +17,13 @@ def __init__(self, *args, **kwargs):
     def set_get_interceptor(self, fn):
         self.get_interceptor = fn
 
+    def copy(self):
```

Comment:
This was a bug.
When copied in `ModelV2.forward()`, all tracking-dict properties (_lazy_tensor_dict) are gone and we are left with a plain dict.